packages/roosterjs-content-model-markdown/lib/modelToMarkdown/creators/createMarkdownBlockGroup.ts

```diff
@@ -30,8 +30,10 @@ export function createMarkdownBlockGroup(
             }
             break;
         case 'FormatContainer':
-            markdownString += createMarkdownBlockQuote(blockGroup, newLinePattern, listCounter);
-            break;
+            if (blockGroup.tagName == 'blockquote') {
+                markdownString += createMarkdownBlockQuote(blockGroup, newLinePattern, listCounter);
+                break;
+            }
         default:
```

Comment:
Although it logically works, but it can't pass lint check.

Also would you please add a test case, thanks